tests/test_biophysical/test_animal/test_cow_calf/test_beef_cow_calf_calculator.py

```
@@ -0,0 +1,802 @@
+    def test_cc_gest_3_late_gestation_higher_than_mid(self) -> None:
+        """Late gestation (DP=250) must have higher NEy than mid-gestation (DP=150)."""
+        mid_inputs = dataclasses.replace(_ANGUS_COW_GEST, days_pregnant=150)
+        late_inputs = dataclasses.replace(_ANGUS_COW_GEST, days_pregnant=250)
+        mid = BeefCowCalfRequirementsCalculator.calculate_requirements(mid_inputs)
+        late = BeefCowCalfRequirementsCalculator.calculate_requirements(late_inputs)
+        assert late.pregnancy_energy > mid.pregnancy_energy
+
+    def test_i4_eqsbw_subtracts_conceptus_weight_for_pregnant_cow(self) -> None:
+        """I4: EQSBW for pregnant cow (DP=200) must be lower than for non-pregnant same BW.
+
+        EQSBW = (SBW - CW) × (SRW/MSBW); CW > 0 → EQSBW_preg < EQSBW_nonpreg.
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)